tests/PHPStan/Analyser/LegacyNodeScopeResolverTest.php

```diff
@@ -5816,7 +5816,7 @@ public static function dataIterable(): array
 				'$iterableWithoutTypehint[0]',
 			],
 			[
-				'iterable',
+				'array|Traversable',
```

Comment:
If `array|Traversable` and `iterable` are the same, wouldn't it better to specify the type to `iterable` instead in order to produce less change on existing resolved type (and avoid modifying existing error messages).

That would say we should use
```
private function getForeachType(): Type
	{
		return new IterableType(new MixedType(), new MixedType());
	}
```
?